server.go

```diff
@@ -1,28 +1,39 @@
 package main
 
-import(
-    "fmt"
-    "github.com/labstack/echo"
-    mw "github.com/labstack/echo/middleware"
+import (
+	"./middleware/auth"
```

Comment:
I cant manage to make it work in docker. For some strange reason it throws an error on docker build. It works fine if I run in my local env.